fcrepo-clients/src/test/java/edu/unc/lib/dl/fcrepo4/PIDsTest.java

```diff
@@ -37,25 +33,14 @@
  */
 public class PIDsTest {
```

Comment:
These tests pass for me in eclipse but not from commandline.  It looks like its a base uri mismatch